TaskOne/ProjectOne/Program.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+
+namespace ProjectOne
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            double x, y, z, result;
+
+            Console.WriteLine("Please enter 3 numbers one by one and press Enter after every number");
+            if (double.TryParse(Console.ReadLine(), out x)) 
+            {
+                if (double.TryParse(Console.ReadLine(), out y))
+                {
+                    if (double.TryParse(Console.ReadLine(), out z))
+                    {
+                        result = (x + y) * (Math.Pow(z, 2) + 1);
+                        Console.WriteLine($"First expression result is: {result}");
+
+                        if (y > 0)
+                        {
+                            result = (x % z - 1) * Math.Sqrt(y);
+                            Console.WriteLine($"Second expression result is: {result}");
+                        }
+                        else
+                        {
+                            Console.WriteLine("y must be > 0 to evaluate the second expression");
+                        }
+
+                        if (z > 0)
```

Comment:
Statement could be reversed 
if (y <= 0) console writeline, 
Without any else - code would be easier to read